README.md

```diff
@@ -1,17 +1,16 @@
 # Flame Behaviors
 
-[![Very Good Ventures][logo_white]][very_good_ventures_link_dark]
-[![Very Good Ventures][logo_black]][very_good_ventures_link_light]
-
-Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
-
 [![ci][ci_badge]][ci_link]
 [![coverage][coverage_badge]][ci_link]
 [![pub package][pub_badge]][pub_link]
 [![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
 [![License: MIT][license_badge]][license_link]
 [![Powered by Flame][flame_badge_link]]([flame_link])
 
+Flame Behaviors applies separation of concerns to game logic in the form of Entities and Behaviors.
```

Comment:
```suggestion
Flame Behaviors applies separation of concerns to game logic in the form of entities and behaviors.
```

minor but was the capitalization intentional?

Reply:
Well it was inconsistent compared to the rest of the docs, so I decided to do it like this